internal/cli/config/rename.go

```diff
@@ -60,9 +60,17 @@ func RenameBuilder() *cobra.Command {
 	const argsN = 2
 	opts := &RenameOpts{}
 	cmd := &cobra.Command{
-		Use:     "rename <oldName> <newName>",
+		Use:     "rename <oldProfileName> <newProfileName>",
 		Aliases: []string{"mv"},
 		Short:   "Rename a profile.",
+		Example: `Rename a profile called myProfile to testProfile:
+$ mongocli config rename myProfile testProfile`,
+		Annotations: map[string]string{
+			"args":               "oldProfileName",
+			"oldProfileNameDesc": "Name of the profile to rename.",
+			"args2":              "newProfileName",
+		    "newProfileNameDesc": "New name of the profile.",
+		},
```

Comment:
@gssbzn I couldn't remember the right format for adding multiple arguments. Is this right?

Reply:
looking at this, https://github.com/mongodb-labs/cobra2snooty/blob/2b19f41b82ab3dafede68a09a7dd98092f352f3d/options.go#L25-L45
it should be 
```go
"args": "oldProfileName, newProfileName",
"requiredArgs": "oldProfileName, newProfileName",
```
I had even forgotten about the required part